### PR TITLE
Position min margin

### PR DESCRIPTION
**Pull request by bachstatter**
This a new liquidation parameter `minimumPositionMargin`, this is a constant value in USD that can control a minimum amount of required collateral that's *NOT* based on positions size

### CODE REVIEW

File: markets/perps-market/test/integration/Liquidation/Liquidation.margin.test.ts [hunk 1]
```diff
@@ -208,14 +214,84 @@ describe('Liquidation - margin', async () => {
...
+          maintenanceMarginFraction,
+          maxLiquidationLimitAccumulationMultiplier,
+          liquidationRewardRatio,
+          maxSecondsInLiquidationWindow,
+          minimumPositionMargin
+        );
+    });
+    // Changing minimumPositionMargin does not have an affect on available margin
+    it('has correct available margin', async () => {
+      assertBn.equal(await systems().PerpsMarket.getAvailableMargin(2), bn(0));
```
Comment (bachstatter) [inline markets/perps-market/test/integration/Liquidation/Liquidation.margin.test.ts:274]: We have this `getAvailableMargin`, that just looks at the total account collateral value + the account PNL.
So it's missing the liquidations considerations, eg most of the time I will not be able to withdraw what it returns.. 
Might be worth calling this something else, and make `getAvailableMargin` take liquidations concerns into account?

File: markets/perps-market/hardhat.config.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ const config = {
     templates,
   },
   mocha: {
-    timeout: 200000,
+    timeout: 120000,
```
Comment (sunnyvempati) [inline markets/perps-market/hardhat.config.ts:24]: lol

File: markets/perps-market/test/integration/Liquidation/Liquidation.margin.test.ts (hunk 1 above)
Comment (sunnyvempati) [reply markets/perps-market/test/integration/Liquidation/Liquidation.margin.test.ts:274]: the `getAvailableMargin` is what gets checked against your maintenance margin for liquidations.  its not meant to be a value that is withdrawable.  its something to display on the UI to check your margin against your pnls.  maybe a `getWithdrawableAmount` which subtracts out the maintenace margin could be nice to add

**Review by sunnyvempati — APPROVED** [review]
lgtm!

File: markets/perps-market/hardhat.config.ts (hunk 2 above)
Comment (bachstatter) [reply markets/perps-market/hardhat.config.ts:24]: haha wait, I thought I increased it...Will revert this

File: markets/perps-market/test/integration/Liquidation/Liquidation.margin.test.ts (hunk 1 above)
Comment (bachstatter) [reply markets/perps-market/test/integration/Liquidation/Liquidation.margin.test.ts:274]: > the `getAvailableMargin` is what gets checked against your maintenance margin for liquidations. its not meant to be a value that is withdrawable. its something to display on the UI to check your margin against your pnls. maybe a `getWithdrawableAmount` which subtracts out the maintenace margin could be nice to add

Yea that what I want. And I like that name too!

**Pull request merged**